src/controller.ts

```diff
@@ -21,6 +22,15 @@ import {
 function loadItemsAndShowQuickPick() {
   quickPick.loadItems();
   quickPick.show();
+
+  const editor = vscode.window.activeTextEditor;
+  if (
+    shouldQuickPickOpenFromSelection() &&
+    editor !== undefined
+  ) {
+    const { start, end } = editor.selection;
+    quickPick.setText(editor.document.getText(new vscode.Range(start, end)));
+  }
```

Comment:
```suggestion

  const activeEditorOrUndefined = vscode.window.activeTextEditor;
  
  if (controller.shouldSearchSelection(activeEditorOrUndefined)) {
    const activeEditor = activeEditorOrUndefined as vscode.TextEditor;
    
    const { start, end } = activeEditor.selection;
    quickPick.setText(editor.document.getText(new vscode.Range(start, end)));
  }
```

ShouldSearchSelection method be invoked from controller due to necessity to mock it in test correctly.